tox.ini

```diff
@@ -15,13 +15,11 @@ skipsdist=True
 [testenv:python]
 commands=
   coverage run manage.py test seed
-  coveralls
```

Comment:
yes because coveralls can't take more than one report per commit. Other coverage tools like codecov can and merge automagically if we switch off coveralls.